pyproject.toml

```diff
@@ -26,7 +26,9 @@ classifiers = [
     "Topic :: Software Development",
     "Typing :: Typed",
 ]
-include = ["CHANGELOG.md"]
+include = [
+    { path = "extensions/**", format = "sdist" },
```

Comment:
Note to self: we also need to add `Cargo.*` to the sdist, or we can't build the rust code.